src/Centreon/Domain/Monitoring/Interfaces/ResourceServiceInterface.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * Copyright 2005 - 2020 Centreon (https://www.centreon.com/)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+declare(strict_types=1);
+
+namespace Centreon\Domain\Monitoring\Interfaces;
+
+interface ResourceServiceInterface
+{
+    /**
+     * Non-ok status in hard state , not acknowledged & not in downtime
+     */
+    public const STATE_UNHANDLED_PROBLEMS = 'unhandled_problems';
```

Comment:
In my opinion, the interface should not have business logic which is likely to expand in the future. In this case 'unhandled_problems' and 'resources_problems' are business terms that are very likely to change/or add new ones to them. I suggest moving them either to the Model or to the implementation of the interface.

Reply:
these constants are not part of the model because the state is not representing there, the dilemma was between repository or service  interface